README.md

```diff
@@ -158,90 +121,63 @@ Formatting for `PrivateKey` or `Peer` follows the format in [Nekoyume Project RE
 ```
 - [Docker Volume Guide](https://docs.docker.com/engine/reference/commandline/volume_create/)
 
-![Docker Volume Create](https://i.imgur.com/ISgKeLc.png)
-
 #### 3. Run your Docker image with your Docker volume mounted (use -d for detached mode)
 
 <pre>
 $ docker run \
 --detach \
 --volume 9c-volume:/app/data \
-planetariumhq/ninechronicles-headless:latest \
+planetariumhq/ninechronicles-headless:[<TAGNAME>] \
 <a href = "#run" title="NineChronicles Headless options">[NineChronicles Headless Options]</a>
 </pre>
 #### Note)
 
-* If you want to use the same headless options as your Nine Chronicles game client, refer to **`config.json`** under **`%localappdata%\Programs\Nine Chronicles\resources\app`**. Inside **`config.json`**, refer to the following properties for your headless options:
-  - `GeniesisBlockPath`
-  - `MinimumDifficulty`
-  - `StoreType`
-  - `AppProtocolVersion`
-  - `TrustedAppProtocolVersionSigners`
-  - `IceServerStrings`
-  - `PeerStrings`
-  - `NoTrustedStateValidators`
-  - `NoMiner`
-  - `Confirmations`
-  - `NonblockRenderer`
-  - `NonblockRendererQueue`
-  - `Workers`
-* If you are using an [Elastic IP](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) on your AWS instance, you must include the IP as the `--host` option but do not need to include the `--ice-server` option.
-* For mining, make sure to include the `--miner-private-key` option with your private key. Also, include `--libplanet-node` to run the default libplanet node.
-
-![Docker Run](https://i.imgur.com/VlwFybj.png)
+- If you want to use the same headless options as your Nine Chronicles game client, please refer to the `headlessArgs` values in value in https://release.nine-chronicles.com/apv.json.
+- If you are using an [Elastic IP](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) on your AWS instance, you can add the IP address in the `--host` option and not use the `--ice-server` option.
+- For mining, make sure to include the `--miner-private-key` option with your private key.
 
 - [Docker Volumes Usage](https://docs.docker.com/storage/volumes/)
 
-### Building Your Own Docker Image from Your Local
+### Building Your Own Docker Image from Your Local Machine
 
 #### Pre-requisites
 
 - Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
 - Docker Hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
 
-#### 1. Build Docker image with the tag name in [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] format.
+#### 1. Build Docker image with the tag name in `[<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>]` format.
 
 ```
-$ docker build . --tag 9c/9c-headless --build-arg COMMIT=9c-1
-
-Usage: docker build . --tag [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] : [<TAGNAME>] --build-arg COMMIT=[<VERSION_SUFFIX>]
+$ docker build . --tag [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] : [<TAGNAME>] --build-arg COMMIT=[<VERSION_SUFFIX>]
```

Comment:
Is it ok to keep whitespaces around `:` between `[<IMAGE_NAME>]` and `[<TAGNAME>]` ? 
I have no idea how docker parses this tag :thinking:

Reply:
Ah this was just for readability but I should delete the whitespaces to avoid confusion. Thanks for the catch! 🙇